### PR TITLE
Improve documentation for users app

### PR DESCRIPTION
**Pull request by Samwalton9**
Updated and expanded documentation for the `users` app. This PR also includes some minor tidying of unused variables.

Where changes were more than documentation changes or might have been confusing I made individual commits to aid with review.

## Phabricator Ticket
[T264425](https://phabricator.wikimedia.org/T264425)

## How Has This Been Tested?
Tests passed, and did some clicking around locally. I tested the Authorization admin page because I was especially unsure about that code removal.

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/users/groups.py [hunk 1]
```diff
@@ -15,13 +15,29 @@
 
 
 def get_coordinators():
+    """
+    Retrieve all users who are in the Coordinator user group.
+
+    Returns
+    ----------
+    Queryset
+        A queryset of user objects
```
Comment (katydidnot) [inline TWLight/users/groups.py:24]: minor nit: QuerySet instead of Queryset for consistency.

File: TWLight/users/groups.py [hunk 2]
```diff
@@ -15,13 +15,29 @@
 
 
 def get_coordinators():
+    """
+    Retrieve all users who are in the Coordinator user group.
+
+    Returns
+    ----------
+    Queryset
```
Comment (suecarmol) [inline TWLight/users/groups.py:23]:
```suggestion
    QuerySet
```

File: TWLight/users/groups.py [hunk 3]
```diff
@@ -15,13 +15,29 @@
...
         pass
 
 
 def get_restricted():
+    """
+    Retrieve all users who are in the Restricted user group.
+
+    Returns
+    ----------
+    Queryset
```
Comment (suecarmol) [inline TWLight/users/groups.py:38]:
```suggestion
    QuerySet
```

File: TWLight/users/helpers/authorizations.py [hunk 4]
```diff
@@ -10,16 +8,35 @@ def get_all_bundle_authorizations():
     """
     Returns all Bundle authorizations, both active
     and not.
+
+    Returns
+    -------
+    Queryset
```
Comment (suecarmol) [inline TWLight/users/helpers/authorizations.py:14]:
```suggestion
    QuerySet
```

File: TWLight/users/groups.py (hunk 2 above)
Comment (suecarmol) [reply TWLight/users/groups.py:23]: Same nit as Katy, I will add this suggestion and then merge this.

**Review by suecarmol — APPROVED** [review]
Thanks for all of your work adding more documentation in key places in the library. I'll go ahead and merge this once I commit the suggestions below 😄

**Pull request merged**